### DISCUSSION TITLE
Jellyfin audiobook support

### DISCUSSION THREAD

**Discussion by Hooloovoo** [PaulWoitaschek/Voice#1715] [Ideas]
This app looks excellent, thank you! 

I have my audiobooks stored with my other media in Jellyfin:
https://github.com/jellyfin/jellyfin-plugin-bookshelf
It would be awesome to be able to use Voice and have this pull down and sync my audiobook library stored on there. You can play audiobooks when online in the official client, but it is not like using an audiobook-focused app.

I already use Finamp:
https://github.com/jmshrv/finamp
to play and sync my music from Jellyfin on Android, but it is very music focused, rather than audiobook focused. Maybe it would give some ideas for implementing this, though.

**Comment by stevenvergenz** [comment]
I second this! I love Voice, and it has real advantages over Finamp. In particular, if I use it to play music it forgets my book playback position :/ It would be amazing to be able to browse and download my books from Jellyfin to my phone with Voice. And progress sync would be a nice feature too, though less important for me.
